src/components/Dataset/ShareDatasetButton.js

```diff
@@ -75,7 +82,7 @@ export const ShareDatasetButton = ({ datasetId }) => {
               label="Copy"
               primary
               responsive
-              onClick={() => handleClick(shareableLink)}
+              onClick={() => handleCopy(shareableLink)}
```

Comment:
```suggestion
              onClick={onCopyClick}
```

Lets not wrap the function here to keep it clearer